src/wallet.rs

```diff
@@ -0,0 +1,16 @@
+use std::fs;
+
+use jsonwebkey as jwk;
+
+pub fn load_from_file(path: &str) -> jwk::JsonWebKey {
```

Comment:
Since this is a library function, I would rather return `Result` than panic internally.